platforms/documentation/docs/src/docs/userguide/dep-man/04-modeling-features/variant_model.adoc

```diff
@@ -361,7 +361,7 @@ Here is the output of the `outgoingVariants` task on a freshly generated `java-l
 --------------------------------------------------
```

Comment:
Is there some way we could replace this with something that's tested?

I think we can `include:...` stuff like the outgoingVariants.out file.

Reply:
It seems like we can, but that would mean we need a sample that produces this output, which it doesn't seem like we have.

Reply:
Hm. Actually, this output changes with this PR. We should add a proper sample then.

Reply:
Done